### ISSUE TITLE
"Couldn't get any posts"

### ISSUE THREAD

**Issue by DhruvParikh1** [kevinzg/facebook-scraper#1123]
Tried the CLI command that was written in the README.md itself:
`facebook-scraper --filename nintendo_page_posts.csv --pages 10 nintendo`

Terminal:
```
(venv) PS C:\Users\DP1\Desktop\Projects\FacebookPosts> facebook-scraper --filename nintendo_page_posts.csv --pages 10 nintendo
Couldn't get any posts.
```

Yes I have a fresh cookies file. Yes I tried it with a different page called `NintendoAmerica` as well.

How to fix plz
Thx

**Comment by manny5457** [comment]
yes I have the same issue, was this ever resolved? even chatGPT couldn't sort it out, does this actually work?

**Comment by gx288** [comment]
same issue here

**Comment by sagi44222** [comment]
Same here
